client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilterStyles.ts

```diff
@@ -17,13 +16,12 @@ const useStyles = makeStyles((theme: Theme) => ({
         textOverflow: 'ellipsis',
     },
     autocomplete: {
-        marginTop: theme.spacing(1),
-        width: '80%',
+        width: '70%',
         height: '60%',
-        marginBottom: theme.spacing(1),
+         marginBottom: theme.spacing(1),
 
         '& .MuiInputBase-root': {
-            fontSize: '0.7vw'
+            fontSize: '0.5vw'
```

Comment:
why is the font size not the same as the card's font size(line 10)? 
this was my reasoning to write this line in the first place :0

Reply:
My bad...